src/www/system_general.php

```diff
@@ -367,6 +412,26 @@
                 </div>
               </td>
             </tr>
+            <tr>
+              <td><a id="help_for_picture" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Picture"); ?></td>
+              <td id="pict_td">
+<?php           if (!empty($pconfig['picture'])): ?>
```

Comment:
```suggestion
<?php if (!empty($pconfig['picture'])): ?>
```

I prefer this style these days as it makes these pages a little easier to read (template vs content)